src/routes/ArticleDetailPage.jsx

```diff
@@ -1,49 +1,84 @@
 import React, { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
+import Header from "../components/Header";
 import ArticleFinder from "../apis/ArticleFinder";
+import CommentList from "../components/CommentList";
 
 const ArticleDetailPage = () => {
   const { id } = useParams();
   const [selectedArticle, setSelectedArticle] = useState(null);
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
     const fetchData = async () => {
       try {
-        const response = await ArticleFinder.get(`/articles/${id}`);
-        setSelectedArticle(response.data.article);
+        const articleResponsePromise = ArticleFinder.get(`/articles/${id}`);
+        const commentsResponsePromise = ArticleFinder.get(
+          `/articles/${id}/comments`
+        ).catch((error) => {
```

Comment:
This an unusual approach because you're mixing the two ways we can handle errors from our promises with `try...catch` and `Promise.catch`. It's better to stick to one or the other for the sake of consistency.

Option 1:
use try catch, which means you can delete the .catch and replace it with `catch (error) { //error handling }` to handle errors coming from the promises
As well as sticking an await on where you're using ArticleFinder

Option 2:
Remove the try{} and go with the .catch approach